src/plugins/messenger/MessengerPreview/components/MessengerButtonTemplate/MessengerButtonTemplate.tsx

```diff
@@ -1,47 +1,49 @@
-import { IFBMButtonTemplatePayload } from '../../interfaces/ButtonTemplatePayload.interface';
```

Comment:
please revert undesired styling changes. E.g.: This file now has tab stops of 2 spaces, we always use 4

Reply:
fixed